int8 scout
int8 excavator
int8 hauler
int8 team_number
int8 desired_team_task